.github/workflows/pre_merge.yaml

```diff
@@ -96,14 +96,18 @@ jobs:
           - task: "detection"
           - task: "instance_segmentation"
           - task: "semantic_segmentation"
-    name: Integration-Test-${{ matrix.task }}-py310
+    name: Integration-Test-${{ matrix.task }}-py312
     steps:
       - name: Checkout repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - name: Install Python
         uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
         with:
-          python-version: "3.10"
+          python-version: "3.12"
+      - name: Install system build dependencies
+        run: |
+          apt update
```

Comment:
It looks like sudo is required here

Reply:
`sudo` is disabled on CI docker image. I'm trying workarounds without having to do all the docker image rebuild and update.